docs/content/installation.rst

```diff
@@ -10,15 +10,14 @@ Remember -- deepTools are available for **command line usage** as well as for
 Requirements
 -------------
 
-* Python 2.7 or Python 3.x
```

Comment:
Is python 2.7 really supported? @LeilyR

Reply:
We should drop python 2.7 if it's not dropped and also python 3.6.